src/main/resources/io/jenkins/plugins/designlibrary/Select/index.jelly

```diff
@@ -23,48 +23,78 @@ THE SOFTWARE.
 -->
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:s="/lib/samples">
-  <s:sample>
-    <p class="jdl-paragraph">${%title}</p>
+  <s:layout>
+    <s:section title="Simple select" description="${%usage.2}">
+      <s:group>
+        <s:preview>
+          <f:entry title="Fruits">
+            <f:select field="fruit"/>
+          </f:entry>
+        </s:preview>
+        <s:code-panes>
+          <s:code-pane title="Jelly">
+            <s:code file="sample2.jelly"/>
+          </s:code-pane>
+          <s:code-pane title="Java">
+            <s:code language="java" file="Sample.java"/>
+          </s:code-pane>
+        </s:code-panes>
+      </s:group>
+    </s:section>
 
-    <p class="jdl-paragraph">${%usage.1}</p>
-    <form action="submit" method="post">
-      <s:code language="xml" file="sample.jelly" />
-      <f:dropdownDescriptorSelector field="fruit" title="Fruits" descriptors="${it.fruitDescriptors}" />
+    <s:section title="Advanced select" description="${%usage.1}">
```

Comment:
It would be ideal if this example also showed the Java code as well

Reply:
Added :)